network-store-client/src/main/java/com/powsybl/network/store/client/BufferedRestNetworkStoreClient.java

```diff
@@ -462,6 +463,14 @@ public int getDanglingLineCount(UUID networkUuid) {
         return client.getDanglingLineCount(networkUuid);
     }
 
+    @Override
+    public void removeDanglingLine(UUID networkUuid, String danglingLineId) {
+        List<Resource<DanglingLineAttributes>> dlToRemove =
```

Comment:
We might have performance issue with this. We can ket it as it is but maybe we will need to change to resources storage data structure for fast removal.